script.js

```diff
@@ -35,29 +35,16 @@ var list = [
 	/you won['’]?t believe/i
 ];
 
-function test( string ) {
-
+function isClickbait( string ) {
 	if ( string.length < 20 || string.length > 100 ) return false;
-
-	for ( var i = 0, l = list.length; i < l; i ++ ) {
-
-		if ( list[ i ].test( string ) ) return true;
-
-	}
-
-	return false;
-
+	return list.some( clickbait => clickbait.test( string ) );
```

Comment:
Actually, I guess this is still in "dev" mode, and currently my local version looks like this:

```js
function test( string ) {

	if ( string.length < 20 || string.length > 100 ) return false;

	for ( var i = 0, l = list.length; i < l; i ++ ) {

		if ( list[ i ].test( string ) ) {

			console.log( i, string );

			return true;

		}

	}

	return false;

}
```